src/Snow.js

```diff
@@ -1,31 +1,72 @@
 import React, { Component } from 'react';
 import {
   StyleSheet,
+  Text,
   View,
   Dimensions,
 } from 'react-native';
+import { oneOf, shape, string } from 'prop-types';
 import Snowflake from './Snowflake';
 
 const windowHeight = Dimensions.get('window').height + (Dimensions.get('window').height * .1);
 
+const lightSnowflakes = [
+  { size: 14, offset: '1%', fallDelay: 0, shakeDelay: 0, }, { size: 24, offset: '5%', fallDelay: 1000, shakeDelay: 1000, },
```

Comment:
I like the idea of different snow configs (strategies).  Please format these in a single column of object rows (one object per row).  Then split them out into separate files, and import them into this file (you can keep existing variable names).  That will keep everything orderly.  Also, the `glyph` prop is missing from this config, please add it.